hack/lib/init.sh

```diff
@@ -33,6 +35,8 @@ github.com/openshift/api/authorization/v1,\
 github.com/openshift/api/build/v1,\
 github.com/openshift/api/config/v1,\
 github.com/openshift/api/image/v1,\
+github.com/openshift/api/osin/v1,\
+github.com/openshift/api/kubecontrolplane/v1,\
```

Comment:
v1 as first version of the group is ambitious.

Reply:
i'm not sure how much it is worth to drive the self-promotion for ourself and deal with different versions... i would like to avoid/defer that pain if possible (even if that means we will say this is 'v1' ;-)

Reply:
> i'm not sure how much it is worth to drive the self-promotion for ourself and deal with different versions... i would like to avoid/defer that pain if possible (even if that means we will say this is 'v1' ;-)

I agree.